staging/src/k8s.io/apimachinery/pkg/apis/meta/v1beta1/register.go

```diff
@@ -33,12 +32,6 @@ func Kind(kind string) schema.GroupKind {
 	return SchemeGroupVersion.WithKind(kind).GroupKind()
 }
 
-// scheme is the registry for the common types that adhere to the meta v1beta1 API spec.
-var scheme = runtime.NewScheme()
-
-// ParameterCodec knows about query parameters used with the meta v1beta1 API spec.
-var ParameterCodec = runtime.NewParameterCodec(scheme)
```

Comment:
No one was using the ParameterCodec here?

Reply:
From v1beta1 - noone in k/k repo and anywhere else that I managed to check.